include/SFML/Graphics/RenderWindow.hpp

```diff
@@ -112,6 +114,19 @@ class SFML_GRAPHICS_API RenderWindow : public Window, public RenderTarget
     ////////////////////////////////////////////////////////////
     Vector2u getSize() const override;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Change the window's icon
+    ///
+    /// The OS default icon is used by default.
+    ///
+    /// \param icon Image to use as the icon. The image is copied,
+    ///             so you need not keep the source alive after
+    ///             calling this function.
+    ///
+    ////////////////////////////////////////////////////////////
+    void setIcon(const sf::Image& icon);
+    using Window::setIcon;
```

Comment:
What does this `using` alias do?

Reply:
By default, an overload in a derived class overshadows a function of the same name in a base class (name hiding). This just imports it back into the current scope, so that both versions can be used (keeping the api a bit more consistent between an `sf::Window` and an `sf::RenderWindow`)

Reply:
https://godbolt.org/z/7zYeMeGjx

Interesting. I did know that.